docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,66 @@
+==============
+Atop connector
+==============
+
+Overview
+--------
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System & Process Monitor) Linux server performance analysis tool.
+
+In order to use this connector, the machine on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+Connector configuration
+-----------------------
+
+Create ``etc/catalog/atop.properties``
+
+.. code-block:: text
+
+    connector.name=atop
+    atop.executable-path=xxx
+
+Configuration variables
+-----------------------
+
+================================================ ====================== ========== =====================================================================================================
+Property Name                                    Default Value          Required   Description
+================================================ ====================== ========== =====================================================================================================
+``atop.concurrent-readers-per-node``             ``1``                  Yes        The number of concurrent read operations allowed per node.
+``atop.executable-path``                         (none)                 Yes        The file path on the local file system for the ``atop`` utility
+``atop.executable-read-timeout``                 ``1ms``                Yes        The timeout when reading from the atop process.
+``atop.max-history-days``                        ``30``                 Yes        The maximum number of days behind to take into account statistics.
+``atop.security``                                ``ALLOW_ALL``          Yes        The authorization rules for the connector. If set to ``FILE``, see :doc:`file-based rules </security/file-system-access-control>`.
+``atop.time-zone``                               System default         Yes        The timezone ID in which the atop data was collected. Generally the timezone of the host.
+================================================ ====================== ========== =====================================================================================================
+
+Usage
+-----
+
+The Atop connector provides a ``default`` schema.
+
+The tables available exposed by this connector can be retrieved by running ``SHOW TABLES``::
+
+    SHOW TABLES in atop.default;
```

Comment:
I think it's FROM: https://trino.io/docs/current/sql/show-tables.html
```suggestion
    SHOW TABLES FROM atop.default;
```

Reply:
both works .. but uppercase for key words..

Reply:
and FROM is probably better ..